internal/providers/azure/pke/workflow/update_cluster.go

```diff
@@ -251,6 +251,20 @@ func UpdateClusterWorkflow(ctx workflow.Context, input UpdateClusterWorkflowInpu
 			return err
 		}
 	}
+	// install node pool labels
+	{
+		activityInput := cluster.RunPostHookActivityInput{
+			ClusterID: input.ClusterID,
+			HookName:  pkgCluster.InstallNodePoolLabelSetOperator,
```

Comment:
This is wrong. We should call `SetupNodePoolLabelsSet` and pass it the appropriate `HookParam`